standard/info/wikis/clashroyale/info.lua

```diff
@@ -28,4 +28,5 @@ return {
 	defaultGame = 'cr',
 	defaultTeamLogo = 'Clash Royale.png', ---@deprecated
 	defaultTeamLogoDark = 'Clash Royale.png', ---@deprecated
+	match2 = 0,
```

Comment:
they have match2
iirc still open prs and also only partially in use
not sure if 0 or 1 is better here

Reply:
I'm gonna go with 0 until it's ✅